tests/src/test/java/alluxio/client/cli/fs/command/PersistCommandTest.java

```diff
@@ -259,16 +263,42 @@ public void persistParallelFilesGtParallelism() throws Exception {
     persistParallel(2, 1);
   }
 
-  public void persistParallel(int totalFiles, int parallelism) throws Exception {
-    int fileSize = 30;
+  @Test
+  public void persistShortTimeout() throws Exception {
+    createUnpersistedFiles("/testPersistTimeout", 100, 50);
+    mFsShell.run("persist", "--timeout", String.valueOf(1), "/*"); // 1ms persist timeout
+    assertTrue("Should log at least one timeout",
+        mLogRule.wasLogged("Timed out waiting for file to be persisted:"));
+  }
+
+  @Test
+  public void persistLongTimeout() throws Exception {
+    int fileSize = 100;
+    List<AlluxioURI> files = createUnpersistedFiles("/testPersistTimeout", fileSize, 25);
+    int ret = mFsShell.run("persist", "--timeout", String.valueOf(60 * 1000), "/*");
```

Comment:
can we add a test for the short version ("-t" instead of "--timeout")?

Reply:
Added unit tests for `getIntArg` and for `PersistCommand` in 87224f3bfea9ea720615480bb9c090c2c6a6dfae